app/boards/arm/ckp/Kconfig.defconfig

```
@@ -0,0 +1,58 @@
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+if BOARD_BT60_V2
+
+config BOARD
+	default "bt60_v2"
+
+config ZMK_KEYBOARD_NAME
+	default "BT60 V2"
+
+endif # BOARD_BT60_V2
+
+if BOARD_BT65_V1
+
+config BOARD
+	default "bt65_v1"
+
+config ZMK_KEYBOARD_NAME
+	default "BT65"
+
+endif # BOARD_BT65_V1
+
+if BOARD_BT75_V1
+
+config BOARD
+	default "bt75_v1"
+
+config ZMK_KEYBOARD_NAME
+	default "BT75"
+
+endif # BOARD_BT75_V1
```

This is certainly our convention, but given the duplication, you might try:

```suggestion
config BOARD
	default "bt60_v2" if BOARD_BT60_V2
	default "bt65_v1" if BOARD_BT65_V1
	default "bt75_v1" if BOARD_BT75_V1

config ZMK_KEYBOARD_NAME
	default "BT60 V2" if BOARD_BT60_V2
	default "BT65" if BOARD_BT65_V1
	default "BT75" if BOARD_BT75_V1
```